backend/src/test/java/quarano/actions/QuarantineEndCheckerTest.java

```diff
@@ -68,7 +76,10 @@ void setUp() {
 		}));
 	}
 
-	@Test // CORE-70
+	/**
+	 * CORE-70
+	 */
+	@Test
```

Comment:
`@Test // CORE-70` is not a valid JavaDoc format and my code formatter always messed up this part).